src/pathways/pathway/pathways.controller.ts

```
@@ -42,6 +42,10 @@ import { AssignPathwayDto } from './dto/assign-pathway.dto';
 import { BulkUpdateOrderDto } from './dto/update-pathway-order.dto';
 import { ActivePathwayDto } from './dto/active-pathway.dto';
 import { ListPathwayUsersDto } from './dto/list-pathway-users.dto';
+import { CheckEligibilityDto } from './dto/check-eligibility.dto';
+import { UpdateHistoryStatusDto } from './dto/update-history-status.dto';
+import { CourseCompletionWebhookDto } from './dto/course-completion-webhook.dto';
+import { PathwayType } from './entities/pathway.entity';
```

ESLint

[error] Replace `'./entities/pathway.entity'` with `"./entities/pathway.entity"`

(prettier/prettier)